client/app.js

```diff
@@ -148,6 +148,183 @@ tie.constant('PREREQ_CHECK_TYPE_BAD_IMPORT', 'badImport');
  * @constant
  */
 tie.constant('PREREQ_CHECK_TYPE_GLOBAL_CODE', 'globalCode');
+tie.constant('PREREQ_CHECK_TYPE_WRONG_LANG', 'wrongLang');
+
+/**
+ * Dictionary of wrong language detection errors and their related information
+ *
+ * @type {{}}
+ */
+tie.constant('WRONG_LANGUAGE_ERRORS', {
```

Comment:
Which file would I put the test under? I'm not sure where it would be most appropriate since it doesn't directly relate specifically to any one object.